cluster/gce/gci/configure-helper.sh

```diff
@@ -224,6 +224,9 @@ function create-master-auth {
   fi
   local -r basic_auth_csv="${auth_dir}/basic_auth.csv"
   if [[ -n "${KUBE_PASSWORD:-}" && -n "${KUBE_USER:-}" ]]; then
+    if [[ -e "${basic_auth_csv}" ]]; then
+      sed -i '/admin,system:masters$/d' "${basic_auth_csv}"
```

Comment:
On GCE/GKE the canonical source for the admin password should be the metadata server and the on disk file is a cache of that value. So the old password is almost exclusively a security vulnerability. That having been said there may be customers who have built something on top so I will add a flag to indicate the canonical source.